src/devices/Board/tests/RpiBoardDtOverlayTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Iot.Device.Board.Tests
+{
+    /// <summary>
+    /// Tests for the dtoverlay file for Raspberry Pi.
+    /// </summary>
+    public class RpiBoardDtOverlayTests
+    {
+        private readonly RaspberryPiBoard board;
+
+        public RpiBoardDtOverlayTests()
+        {
+            board = new RaspberryPiBoard();
+        }
+
+        [Fact]
+        public void CheckI2cConfig()
+        {
+            board.ConfigurationFile = Path.Combine("ConfigFiles", "config.I2c.txt");
```

Comment:
do we also want to add actual tests for System.Device.Gpio.Tests? You could do similar test but use default path (make sure to add `[Trait(...)]` attribute similar to what there already is for PWM/SPI/I2C tests

Reply:
I added a specific test that output the configuration. It will always pass, it's not intended to fail, just outputting the configuration. I guess it's a good compromise

Reply:
We can once merged find the best way to use it in the other tests